### PR TITLE
add metadata to MONDO:0010778

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
addresses #1085

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -210329,7 +210330,9 @@ xref: OMIM:500007 {source="MONDO:equivalentTo"}
 xref: UMLS:C0152164 {source="NCBI:mim2gene_medline", source="MONDO:notFoundInDiseaseSubset", source="OMIM:500007"}
 xref: UMLS:C1838991 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:500007"}
 xref: UMLS:C1838992 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:500007"}
-is_a: MONDO:0003847 ! inherited genetic disease
+is_a: MONDO:0003847 {source="OMIM:500007"} ! inherited genetic disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: are we sure?

**Comment by cmungall** [comment]
oops, conflicts

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: OMIM says inheritance is:
- Mitochondrial
- Multifactorial

https://www.omim.org/entry/500007

Should we make 'inherited genetic disease' an excluded subclass of?

**Comment by nicolevasilevsky** [comment]
I fixed the conflicts

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (cmungall) [reply src/ontology/mondo-edit.obo]: I think just get rid of the axiom. We only record historic if it another source asserts it to be the case.

But we do need to be clearer about our definition of inherited genetic disease

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: ok, removed the superclass assertion to inherited genetic disease

**Pull request merged**